hubmap/frontend/cypress/integration/live_auth_test.js

```diff
@@ -0,0 +1,63 @@
+// ideally it is not recommended to test tags/redirects outside of domain in cypress. we should not be testing this in cypress,
```

Comment:
- As discussed, we do not want to check in code that does not run: This file should be removed.
- Preference for capitalizing sentences.

Reply:
ohh yes sure, I can delete this. later whenever you guys want to have prod tests, you can always refer this. 

> * As discussed, we do not want to check in code that does not run: This file should be removed.
> * Preference for capitalizing sentences.

oh yes, sure I will delete this, later when you/anyone decide to have prod tests, you can always refer /this discussion can be helpful.